.travis.yml

```diff
@@ -25,16 +25,16 @@ jobs:
     name: "Run tests gRPC example Java"
 
   - script:
-    - "bin/runSbtJob shopping-cart/shopping-cart-scala test"
-    name: "Run tests gRPC example Scala"
+    - "bin/runSbtJob grpc-example/grpc-example-scala test"
+    name: "Run tests gRPC example Scala"    
 
   - script:
-    - "bin/runSbtJob shopping-cart/shopping-cart-java test"
-    name: "Run tests gRPC example Java"
+    - "bin/runSbtJob shopping-cart/shopping-cart-scala test"
+    name: "Run tests Shopping Cart example Scala"
 
   - script:
-    - "bin/runSbtJob grpc-example/grpc-example-scala test"
-    name: "Run tests gRPC example Scala"    
+    - "bin/runSbtJob shopping-cart/shopping-cart-java test"
+    name: "Run tests Shopping Cart example Java"
```

Comment:
Shopping cart seems to break the convention to put Java before Scala.

Reply:
My mistake 😞 Fixed 😄